tmt/cli/_root.py

```diff
@@ -1620,6 +1620,62 @@ def _construct_trying_provision_options(params: Any) -> dict[str, Any]:
     return options
 
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Status
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@main.command()
+@pass_context
+@workdir_root_options
+@option(
+    '-i', '--id', metavar="ID", multiple=True,
+    help='Run id(name or directory path) to show status of. Can be specified multiple times.')
+@option(
+    '--abandoned', is_flag=True, default=False,
+    help='List runs which have provision step completed but finish step not yet done.')
+@option(
+    '--active', is_flag=True, default=False,
+    help='List runs where at least one of the enabled steps has not been finished.')
+@option(
+    '--finished', is_flag=True, default=False,
+    help='List all runs which have all enabled steps completed.')
+@verbosity_options
+def status(
```

Comment:
The status definition has been moved to a separate file `status.py`. Please, move the changes there.

Reply:
> The status definition has been moved to a separate file status.py

Yeah,that's why I updated in status.py to fix the error, but I forgot to check the whole change,so failing to note we still have status in _root.py :sweat_smile:  Anyway,addressed:https://github.com/teemtee/tmt/pull/2850/commits/abe8599734da0cba6c7c5a8c6654bc0311399ebd

Reply:
Great, thanks!